crawlerr/services/rag_service.py

```
@@ -341,6 +586,86 @@ async def _rerank_results(
+            
+            # Sort by updated scores
+            matches.sort(key=lambda m: m.score, reverse=True)
+            
+            # Return top_k if specified
+            if top_k and top_k < len(matches):
+                matches = matches[:top_k]
+            
+            logger.debug(f"Qwen3 reranked {len(matches)} results")
+            return matches
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)